docs-website/src/pages/docs/_components/QuickstartCards/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import clsx from "clsx";
+import styles from "./quickstartcards.module.scss";
+import QuickstartCard from '../QuickstartCard'
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import Link from "@docusaurus/Link";
+import {
+  ThunderboltTwoTone,
+  ApiTwoTone,
+  DeploymentUnitOutlined,
+  SyncOutlined,
+  CodeTwoTone,
+  QuestionCircleTwoTone,
+  SlidersTwoTone,
+  HeartTwoTone,
+} from "@ant-design/icons";
+
+const quickstartContent =  [
+{
+    title: "Quickstart with DataHub",
+    icon: "datahub-logo-color-mark",
+    to: "docs/quickstart",
```

Comment:
it seems like these links doesn't respect the currently active version

there might be some weird docusaurus magic required here to prefix the links (https://docusaurus.io/docs/docusaurus-core#useDocusaurusContext). alternatively, it might work if you make them relative links e.g `./quickstart` or something